### PR TITLE
Write down tips for using OBS

### LINKED ISSUE #1341: Document how to work with OBS for Linux builds
While working with OBS I've learned a bunch of non-obvious and undocumented things that are important to know. This information should be recorded in RD docs (within this repo).

### PR DESCRIPTION
**Pull request by adamkpickering**
Closes #1341.

### CODE REVIEW

File: docs/development/obs.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+too slow, you can tell `osc build` to only fetch packages from the
+build.opensuse.org api with the `--download-api-only` flag.
+
+### How to skip running services before build
+
+Use the `--no-service` flag on `osc build` for this.
+
+### How to find the output of a local build
+
+When you build locally, the output of the build will go to a very obscure
```
Comment (ericpromislow) [inline docs/development/obs.md:126]: No need for `very`

**Review by ericpromislow — CHANGES_REQUESTED** [review]
Needs tweaks

File: docs/development/obs.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+endpoint from which you can download packages. There are several types of
+repositories - some are true repositories in the sense that tools like
+`apt` and `dnf` can be configured to use them, and others are just endpoints
+you can download assets from. Also, you can configure multiple repositories
+on each project. This is useful for building and serving packages of multiple
+formats from the same binary or source code.
+
+A **package** is also configured on a project. Conceptually, OBS packages
+are different from packages in other contexts. In OBS, a package represents
+a set of files that goes into a build, such as source files and any package
```
Comment (ericpromislow) [inline docs/development/obs.md:51]: 'set of files that go into a build' sounds better

File: docs/development/obs.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+
+## Important Concepts
+
+There are a few concepts that one should understand in order to use OBS.
+The way they work and interact can be unintuitive at first, so a brief
+overview is provided here.
+
+A **project** is the object in which you do everything in OBS.
+Everything falls under projects: repositories, packages, services,
```
Comment (ericpromislow) [inline docs/development/obs.md:31]: `services;` (since the next phrase isn't a continuation)

**Review by ericpromislow — CHANGES_REQUESTED** [review]
Just minor change requests.

Note that I don't have a SUSE machine at hand to try this with. Someone else should verify the commands work (maybe?).

**Comment by adamkpickering** [comment]
> Just minor change requests.
> 
> Note that I don't have a SUSE machine at hand to try this with. Someone else should verify the commands work (maybe?).

I made some more changes. Regarding the commands, they're quite simple and I've run them myself (many times in some cases). Plus this is a docs PR - it's not worth much more time IMO. But if you want to wait for someone else to give them a shot that's fine by me.

**Review by ericpromislow — APPROVED** [review]

**Pull request merged**